app_dart/lib/src/model/appengine/task.dart

```diff
@@ -42,6 +42,7 @@ class Task extends Model<int> {
     this.buildNumberList,
     this.builderName,
     this.luciBucket,
+    this.isBenchmark,
```

Comment:
Let's define a default `false`, which should be okay considering old data is null. (front end is based on tot task status).